src/api/index.ts

```diff
@@ -429,7 +429,11 @@ router.post(
       res.send('Successfully called migrationComplete');
       await subscription.update({ syncStatus: 'COMPLETE' });
 
-      statsd.increment(metricSyncStatus.complete);
+      const tags = {
+        installationId: githubInstallationId.toString()
```

Comment:
Unfortunately, we can't use "InstallationID" as a tag, because we'll explode tags cardinality.

Tags should have limited amount of variants. For example "response code" is OK, beause we are generally getting not more of dozen of them. InstallationID or Jira Host name are bad, because we have tens of thousands of Jira Instances connected to GitHub